ocfweb/docs/docs/services/hpc/slurm.md

```diff
@@ -0,0 +1,142 @@
+[[!meta title="Slurm"]]
+
+## Slurm
+
+In order to use the resources of our HPC computng nodes, you must submit your
+computing tasks through [Slurm][slurm], which will ensure that your task, or
+job, is given exclusive access to some CPUs, memory, and GPUs if needed. Slurm
+also intelligently queues jobs from different users to most efficiently use our
+nodes' resources.
+
+## Running a job through Slurm
+
+Submitting a job to Slurm can be done in one of two ways: through `srun`, and
+through `sbatch`.
+
+When using `srun`, options are supplied using command-line flags, and the job
+is attached to the terminal you run `srun` from. If there is a queue to run
+jobs, the terminal will wait until your job starts running, and if the terminal
+closes, the job will be cancelled.
+
+By contrast, to submit a job using `sbatch`, you must first create a batch file
+that includes options for your job, and the commands that your job will run.
+The batch file is then submitted, and after the job runs, log files with the
+job's output to `stdout` are put into your home directory.
+
+#### When to use `srun` versus `sbatch`
+
+If your job requires interactivity or inputs from the terminal, or you need a
+terminal to test or experiment, use `srun`. Otherwise, use `sbatch`, as you
+don't have to keep your terminal open until the job runs.
+
```

Comment:
Could mention that even noninteractive programs can use `srun` with `tmux` or `screen`.